### ISSUE TITLE
MyQ not working in 0.92.1

### ISSUE THREAD

**Issue by gorobotgo** [home-assistant/hassio#1069]
**Home Assistant release with the issue:**
0.92.1

**Operating environment (HassOS/Generic):**
hass.io, Raspberry Pi

**Supervisor logs:**
```
2019-05-01 17:04:05 ERROR (MainThread) [homeassistant.components.cover] Error while setting up platform myq
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/homeassistant/helpers/entity_platform.py", line 126, in _async_setup_platform
    SLOW_SETUP_MAX_WAIT, loop=hass.loop)
  File "/usr/local/lib/python3.7/asyncio/tasks.py", line 416, in wait_for
    return fut.result()
  File "/usr/local/lib/python3.7/site-packages/homeassistant/components/myq/cover.py", line 43, in async_setup_platform
    myq = await login(username, password, brand, websession)
  File "/usr/local/lib/python3.7/site-packages/pymyq/api.py", line 291, in login
    await api.authenticate(username, password)
  File "/usr/local/lib/python3.7/site-packages/pymyq/api.py", line 226, in authenticate
    await self._get_security_token()
  File "/usr/local/lib/python3.7/site-packages/pymyq/api.py", line 242, in _get_security_token
    login_request=True,
  File "/usr/local/lib/python3.7/site-packages/pymyq/api.py", line 151, in _request
    return await resp.json(content_type=None)
  File "/usr/local/lib/python3.7/site-packages/aiohttp/client_reqrep.py", line 1036, in json
    return loads(stripped.decode(encoding))
  File "/usr/local/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Description of problem:**
myq fails to load at startup after upgrading to 0.92.1 from 0.91.3.

**Comment by gorobotgo** [comment]
In case this is potentially an issue with my config, here's the relevant section from my configuration.yaml:

```
cover:
  - platform: myq
    username: my@email.com
    password: myPassword
    type: chamberlain
```

This had been working for me for quite some time now, though, and it's a match for the config example in the documentation, and I haven't been able to find mention of breaking changes around myq in the last few releases, so I'm relatively convinced that this isn't the culprit.

**Comment by pvizeli** [comment]
Look like the response from server is not a valid json

**Comment by gorobotgo** [comment]
This appears to have been resolved in 0.92.2:

Fix authentication issue (@ehendrix23 - #23600)

Confirmed that it's working for me after updating.
